common/primitives/src/msa.rs

```diff
@@ -27,31 +27,21 @@ impl From<Delegator> for MessageSourceId {
 
 /// KeyInfo holds the information on the relationship between a key and an MSA
```

Comment:
Updates in this file is simply removing Blocknumber which was used in expiry which is now deleted